test/gateway/eip20_gateway/stake.js

```diff
@@ -317,5 +318,24 @@ contract('EIP20Gateway.stake() ', function (accounts) {
         errorMessage = "Previous process is not completed";
         await _stake(utils.ResultType.FAIL);
     });
-});
 
+    it('should fail stake if gateway is not activated.', async function () {
+
+        let mockToken = await MockToken.new();
+        let baseToken = await MockToken.new();
+        let bountyAmount = new BN(100);
+
+        gateway = await Gateway.new(
+            mockToken.address,
+            baseToken.address,
+            accounts[1], //core address
+            bountyAmount,
+            accounts[2] // organisation address
+        );
+
+        await _setup(accounts, gateway);
+        await _prepareData();
+        await _stake(utils.ResultType.FAIL);
```

Comment:
Where is the gateway activated in the other test cases? 😕

Reply:
In the mock contract, it has been set to` true` on construction.
[Link](https://github.com/sarvesh-ost/mosaic-contracts/blob/daa109f14639a7677867e206fab85c0b3ee6a3e6/contracts/test/TestEIP20Gateway.sol#L58)